code/game/objects/items/weapons/bee_briefcase.dm

```diff
@@ -0,0 +1,91 @@
+
+/obj/item/weapon/bee_briefcase
+	name = "briefcase"
+	desc = "This briefcase has easy-release clasps and smells vaguely of honey and blood..."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "briefcase"
+	item_state = "briefcase"
+	flags = CONDUCT
+	hitsound = "swing_hit"
+	force = 10.0
```

Comment:
you'd be surprised how many items have these decimals, though as you said they are unnecessary. they also don't break anything, so it doesn't make sense to make a commit just to remove them